spec/support/capybara.rb

```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-Capybara.default_driver = :selenium_chrome_headless
+Capybara.default_driver = :selenium_chrome
```

Comment:
We'll probably want to keep this as chrome_headless for CI purposes. I'm also okay with a tenary operator here based on if we're in CI or not, effectively `ENV['CI'] ? :selenium_chrome_headless : :selenium_chrome`

Reply:
This is why CI is failing at the moment on this branch